src/main/webapp/video_context.js

```diff
@@ -30,9 +30,12 @@ async function getVideoContext(urlInput) {
  */
 async function updateUIWithVideoContext(url) {
   clearElement('video-context');
+  clearElement('video-embed');
   videoContext = await getVideoContext(url);
   document.getElementById('video-context').innerHTML =
   videoContextAsHTML(videoContext);
+  document.getElementById('video-embed').innerHTML = '<iframe width="500" height="300"src=' + 'https://www.youtube.com/embed/'+ extractYouTubeUrl(url)+ '?controls=0;' +
```

Comment:
I'm confused why are we embedding the video twice?

This may result in a strange experience since it will mean that if the user was watching the embedded video before clicking "submit" from the search results menu, their progress will be lost.

Reply:
I am guessing it is a different tab so they want to load a different video frame here?

Reply:
With the way it is done, we are actually embedding more than 2 videos, as each searched video will have this.

Our current js files organization is not great - script.js has a bunch of random methods that are used all over the place. We may want to think of a better way to organize all the methods. For example, all functions that will be called more than once should go into comm_utils.js, and we should rename script.js to make it more meaningful.